client/src/store/utils/thunkCreators.js

```diff
@@ -111,6 +119,25 @@ export const postMessage = (body) => async (dispatch) => {
   }
 };
 
+const updateReadStatus = async (unreadMessages) => {
+  const statusPromises = unreadMessages.map(messageId=>{
+      return axios.put(`/api/messages/${messageId}`, {read: true});
+  });
```

Comment:
Suggestion: instead of sending multiple requests to `/api/messages/:messageId`, how about creating a new route that updates all messages in a conversation?

Reply:
done